src/main/java/core/org/akaza/openclinica/service/crfdata/EnketoUrlService.java

```diff
@@ -656,7 +662,11 @@ public File getFormPdf(String subjectContextKey, PFormCacheSubjectContextEntry s
         crfOid = formLayout.getOcOid() + DASH + formLayout.getXform() + crfFlavor;
 
         // Call Enketo api to get url
-        EnketoAPI enketo = new EnketoAPI(EnketoCredentials.getInstance(studyOid));
+        EnketoCredentials enketoCredentials = EnketoCredentials.getInstance(studyOid);
+        if(enketoCredentials.getServerPdfUrl() != null) {
+        	enketoCredentials.setServerUrl(enketoCredentials.getServerPdfUrl());
+        }
+        EnketoAPI enketo = new EnketoAPI(enketoCredentials);
```

Comment:
Move this logic to EnketoCredentials#getPdfInstance and remove serverPdfUrl field from that class.